Sources/SwiftFormat/Core/WithAttributesSyntax+Convenience.swift

```diff
@@ -13,16 +13,23 @@
 import SwiftSyntax
 
 extension WithAttributesSyntax {
-    /// Indicates whether the node has attribute with the given `name`.
-    ///
-    /// - Parameter name: The name of the attribute to lookup.
-    /// - Returns: True if the node has an attribute with the given `name`, otherwise false.
-    func hasAttribute(_ name: String) -> Bool {
-        attributes.contains { attribute in
-            let attributeName = attribute.as(AttributeSyntax.self)?.attributeName
-            return attributeName?.as(IdentifierTypeSyntax.self)?.name.text == name
-            // support @Module.Attribute syntax as well
-            || attributeName?.as(MemberTypeSyntax.self)?.name.text == name
-        }
+  /// Indicates whether the node has attribute with the given `name`.
```

Comment:
Can you mention in the doc comment that the module name is only considered if present if the attribute is written as `@Module.Name`, and that that's intentional? Otherwise it might look like a bug to someone reading it in the future.

Reply:
good point, done. 👍